azure/conf.py

```diff
@@ -65,6 +65,8 @@
         "../instances/install-azure-cli/",
     "azure-how-to/find-ubuntu-images":
         "../instances/find-ubuntu-images/",
+    "azure-how-to/launch-ubuntu-images":
+        "../instances/launch-ubuntu-images/",
```

Comment:
This redirection is not needed. The others are needed to account for old paths (when how-to guides where located directly under the azure-how-to folder).

Reply:
Understood, will remove.